e2e/framework/testserver.go

```diff
@@ -538,6 +544,52 @@ func (t *TestServer) WaitForReceipt(ctx context.Context, hash web3.Hash) (*web3.
 	return data.receipt, data.err
 }
 
+// WaitForGasTotal waits for the total gas used sum for the passed in
+// transactions
+func (t *TestServer) WaitForGasTotal(txHashes []web3.Hash) uint64 {
+	var (
```

Comment:
You might want to call `t.t.Helper()` here so that you can check where test failed. If you don't call, test reports a line in this function as error. But if you call this, the line which called this function will be reported when test failed

Reply:
Good catch 💯 

Fixed 👍 

[5f8c638](https://github.com/0xPolygon/polygon-edge/pull/485/commits/5f8c638d1ca97acef59fd8895ba51974adceffd2)